FHEM/lib/SD_ProtocolData.pm

```diff
@@ -2856,7 +2856,7 @@ package lib::SD_ProtocolData;
         sync            => '2DD4',
         modulation      => '2-FSK',
         rfmode          => 'Bresser_5in1',
-        register        => ['0001','0246','0306','042D','05D4','06FF','07C0','0802','0D21','0E65','0FE8','1088','114C','1202','1322','14F8','1551','1916','1B43','1C68'],
+        register        => ['0001','0246','0346','042D','05D4','06FF','07C0','0802','0D21','0E65','0FE8','1088','114C','1202','1322','14F8','1551','1916','1B43','1C68'],
```

Comment:
Das bringe ich gerade mit dem Umbenennen von Readings nicht zusammen.
Ist das ggf. eine andere Änderung die nichts mit den Readings zu tun hat?

Reply:
Das Hauptanliegen des PR war halt das Umbenennen der Readings.
Für die kleine Änderung dieses einen Registerwertes wollte ich mir die Mühe sparen, extra einen PR zu tätigen. In der CHANGED steht diese Änderung schon mit drin und in deinen Abfragekatalog habe ich es jetzt auch noch mit aufgenommen.